### PR TITLE
Attempt to fix dotnet test flakes

### PR DESCRIPTION
**Pull request by rli**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/RunConfigTestUtils.kt [hunk 1]
```diff
@@ -59,7 +59,13 @@ fun executeRunConfiguration(runConfiguration: RunConfiguration, executorId: Stri
             val executionEnvironment = executionEnvironmentBuilder.build()
 
             val listener = object : OutputListener() {
+                override fun processWillTerminate(event: ProcessEvent, willBeDestroyed: Boolean) {
+                    println("Received processWillTerminate for ${event.processHandler}")
```
Comment (manodnyab) [inline jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/RunConfigTestUtils.kt:63]: Do we need to log both processWillTerminate and processTerminated?

Comment (rli) [reply jetbrains-core/tst/software/aws/toolkits/jetbrains/utils/RunConfigTestUtils.kt:63]: trying to understand the lifecycle here and if it's possible for one to be called without the other

File: jetbrains-rider/tst/base/AwsReuseSolutionTestBase.kt [hunk 2]
```diff
@@ -45,6 +49,15 @@ abstract class AwsReuseSolutionTestBase : BaseTestWithSolutionBase() {
         allowCustomDotnetRoots()
     }
 
+    @BeforeClass(alwaysRun = true)
```
Comment (manodnyab) [inline jetbrains-rider/tst/base/AwsReuseSolutionTestBase.kt:52]: Why do we provide this flag explicitly(in the previous After tag as well)? Is the default false?(that's surprising)

**Review by manodnyab — APPROVED** [review]

File: jetbrains-rider/tst/base/AwsReuseSolutionTestBase.kt (hunk 2 above)
Comment (rli) [reply jetbrains-rider/tst/base/AwsReuseSolutionTestBase.kt:52]: testng default behavior is to skip the after* methods if the test fails. for the before* methods this forces the method to be run regardless of what test group it's in

**Pull request merged**